extension/intents/playing/playing.js

```diff
@@ -0,0 +1,33 @@
+/* globals searching */
+
+this.intents.playing = (function() {
+  this.intentRunner.registerIntent("play", async (desc) => {
+    let playerTab;
+    if (desc.slots.query) {
+      // Multi-part execution task: will do magical IFL Google Search, then execute play once the page loads
+      const googleQueryURL = searching.googleSearchUrl(desc.slots.query, true);
+      playerTab = await browser.tabs.create({
+        url: googleQueryURL,
+      });
+    } else {
+      playerTab = await browser.tabs.getCurrent();
+    }
+
+    // get video content for the current tab
+    setTimeout(async () => {
+      await browser.tabs.executeScript(playerTab.id, {
+        file: "/extension/intents/playing/playMedia.js",
```

Comment:
I think the extension/ directory is the root path for everything, so this should be `file: "/intents/playing/playMedia.js",`